test_dagmc_stats.py

```diff
@@ -99,3 +99,11 @@ def test_get_surfaces_per_volume():
     for eh in range(known_volumes.size()):
         surfs = my_core.get_child_meshsets(known_volumes[eh]).size()
         assert(surfs == s_p_v_data[eh])
+
+def test_get_area_triangle():
```

Comment:
This matches the tests for previous functions, but we probably need to start improving the scope of testing.

For example, it would be wise to split out the area function to two pieces: one that gets the data from MOAB and another that uses that data to calculate the area.  Then it would be possible to test the function that calculates the area with a variety of data/cases that are independent of MOAB.

Reply:
We did discuss this issue with the testing. The current geometry used in these tests has too many triangles and volumes to easily figure out which triangle is which and calculate the size of it by hand. I advised Yuehan to make this PR so we could review it first and then make a second PR with a more useable geometry for better testing (I have to make a better geometry file for her still). Given the question of whether the area calculation is correct, it might be worthwhile making sure it gets in with this PR as well.

Reply:
What do you think about separating out the method for calculating the area from the method for getting data from MOAB?

Reply:
Yes, I think that is a good separation to make too. I made a comment (sort of) about this on `dagmc_stats.py`. The separation makes sense for the reason you state here and because of the repeated code between this function and the triangle_aspect_ratio function.